editor/editor_node.cpp

```diff
@@ -510,6 +510,9 @@ void EditorNode::_update_from_settings() {
 	Viewport::MSAA msaa = Viewport::MSAA(int(GLOBAL_GET("rendering/anti_aliasing/quality/msaa_2d")));
 	scene_root->set_msaa_2d(msaa);
 
+	// 2D doesn't use a dedicated SubViewport like 3D does, so we apply it on the root viewport instead.
+	scene_root->set_use_debanding(GLOBAL_GET("rendering/anti_aliasing/quality/use_debanding"));
+
```

Comment:
```suggestion
	bool use_debanding = GLOBAL_GET("rendering/anti_aliasing/quality/use_debanding");
	scene_root->set_use_debanding(use_debanding);
	get_viewport()->set_use_debanding(use_debanding);
```

By changing it to this, it fixes both #108975 and #108948 and should sill be safe for cherry-picking.